homeassistant/components/sensor/ebox.py

```diff
@@ -32,7 +31,7 @@
 DEFAULT_NAME = 'EBox'
 
 REQUESTS_TIMEOUT = 15
-SCAN_INTERVAL = timedelta(minutes=5)
+MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=15)
```

Comment:
Why change this from SCAN_INTERVAL? With SCAN_INTERVAL, Home Assistant will make sure to not call update more than once every 15 minutes per entity.

Reply:
Setting both constants and a throttle is good if you want to have a hard limit on the interval and set a default interval for updates. Users can override `SCAN_INTERVAL` but not the throttle.

Reply:
@balloob that is a data object, so it need a throttle